webapp/app/forms/steps/lotse/steuerminderungen_steps.py

```diff
@@ -19,7 +19,7 @@ class StepSteuerminderungYesNo(FormStep):
 
     class Form(SteuerlotseBaseForm):
         steuerminderung = RadioField(
-            label=_l('form.lotse.field_steuerminderung'),
+            label='',
```

Comment:
Mh, somehow, the label gets set to "Steuerminderung" if I set it to None or leave it completely. That's because of this line in `Field`:

```self.label = Label(self.id, label if label is not None else self.gettext(_name.replace('_', ' ').title()))```
That's too smart for us. :D

Reply:
Thank you. I should have read that more carefully. Maybe you can add a comment then to explain why it's `''`?